### ISSUE TITLE
Looks like things may be breaking now.

### ISSUE THREAD

**Issue by sbromberger** [andrewcooke/ParserCombinator.jl#24]
https://travis-ci.org/JuliaGraphs/LightGraphs.jl/builds/225059949#L431-L456

**Comment by sbromberger** [comment]
Just an FYI - this started breaking between Julia version `daefda40aa` and `6e0a2f8c94` - unfortunately, I don't have time to run a bisect at this time.

**Comment by andrewcooke** [comment]
So should I make a new release where the package version required has an upper limit?

**Comment by sbromberger** [comment]
Nope. it broke due to a bad Julia commit. Stay the course, but you'll probably need to rewrite your tasks with produce/consume at some point.
